hibernate-reactive-core/src/test/java/org/hibernate/reactive/OrderTest.java

```diff
@@ -0,0 +1,242 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.hibernate.metamodel.model.domain.EntityDomainType;
+import org.hibernate.metamodel.model.domain.internal.MappingMetamodelImpl;
+import org.hibernate.query.Order;
+import org.hibernate.reactive.containers.DatabaseConfiguration;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIf;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.junit5.Timeout;
+import io.vertx.junit5.VertxTestContext;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.metamodel.SingularAttribute;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.stream.Collectors.toList;
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.MYSQL;
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.MARIA;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisabledIf( "skipDBs")
+@Timeout(value = 10, timeUnit = MINUTES)
+public class OrderTest extends BaseReactiveTest {
+	final Book book1 = new Book("9781932394153", "Hibernate in Action");
+	final Book book2 = new Book("9781617290459", "Java Persistence with Hibernate");
+
+	@Override
+	protected Collection<Class<?>> annotatedEntities() {
+		return List.of(
+				Book.class
+		);
+	}
+
+	static boolean skipDBs() {
+		// MySQL & MariaDB will fail because the simple 'String isbn' element will prevent book from being created correctly
+		// errorMessage=Table 'hreact.Book' doesn't exist, errorCode=1146
+		// NOTE: ORM's OrderTest will also fail with these DB's
+		return DatabaseConfiguration.dbType() == MYSQL ||
```

Comment:
If I run the test with MySQL it works fine. There was probably something wrong on your machine.

Reply:
FYI... yes.. clearing my docker containers and rebooting... all DB's work